opentelemetry-etw-logs/README.md

```diff
@@ -28,7 +28,51 @@ captured by agents running locally and listening for specific ETW events.
 ## Viewing ETW Logs
 
 Logs exported to ETW can be viewed using tools like `logman`, `perfview` etc.
-// TODO - add instructions.
+
+### Using `logman`
+
+To view the telemetry emitted to ETW you can use [`logman`](https://learn.microsoft.com/windows-server/administration/windows-commands/logman) along with `perfview`.
+`logman` will listen to ETW events from the given provider (on this example, `provider-name`) and store them in a `.etl` file.
```

Comment:
nit: I think its easier to follow if we use "my-provider", "my-event-name" in the examples.